src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+header {
```

Comment:
Use class selectors instead of tag ones
Point 1 [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)